site/app/templates/admin/Docker.twig

```diff
@@ -5,79 +5,141 @@
     {% if docker_info is not null %}
         {# TODO: include worker information here in the future #}
         <p>Docker Version : {{ docker_info['ServerVersion'] }} </p>
+        <p>Updated on: {{last_updated}}</p>
+        {% if error_logs | length > 0 %}
+            <textarea disabled="disabled" style="resize: none; width: 100%; height:25vh;">
+                {% for error_log in error_logs %}
+                    {{error_log ~ "\n"}}
+                {% endfor %}
+            </textarea>
+        {% endif %}
+        <button class="btn btn-primary" id="update-machines" onclick=updateImage("{{admin_url}}/update_docker") style="margin-top: 1em"> Update dockers and machines</button>
     {% endif %}
 </div>
 
 <div class="content">
-    <h2 class="docker-header">Autograding Docker Images</h2>
-    <button class="btn btn-default" 
-            onclick="collapseSection('autograding-images','autograding-images-collapse')"
-            id="autograding-images-collapse">
-            Collapse
-    </button>
-
-    <div id="autograding-images" style="display:block;">
-        <p class="black-message">These are the docker images that are installed and listed under autograding_containers.json</p>
-
-        <div class="docker-images-container">
-            {{ _self.renderDockerImage(autograding_containers['found']) }}
-        </div>
-    </div>
+    <h2 class="docker-header">Worker Machines</h2>
+    <table class="machine-table">
+        <tr>
```

Comment:
Ok I didn't look closely enough at the image of the table you pasted last review, and am changing my mind on adding that extra header - since the content to the right of the "Machine" header is actually data, not headers themselves its confusing that there is an actual column header on the same row.

I'm now thinking something like this better expresses the data you're trying to visualize

<img width="1150" alt="image" src="https://user-images.githubusercontent.com/12129065/127089422-e99e7969-3721-448a-abfa-76e04aa431b7.png">

```html
<table class="machine-table">
    <tbody>
        <tr>
            <th>Machine</th>
            <th colspan="6" style="text-align:center;">Capability</th>
        </tr>
        <tr>
            <th></th>           
            <th>cpp</th>
            <th>default</th>
            <th>et-cetera</th>
       </tr>
```

And you can get the number of cols from twig and maybe the headers don't need to bold for the capibilities? Might look better 

I realize this PR has been open for a while however so this is lower priority can be ignored if there's not enough time or it can be discussed in the next meeting with @bmcutler and others.

Reply:
This is what I have changed it to, does this look okay?
![image](https://user-images.githubusercontent.com/49821332/127169684-bfa5004a-8f13-4ca0-8b26-50d017db4b8d.png)

Reply:
@HenrikLam  I would suggest moving the num autograing works column next to the machine column and then having the capabilities to the right so the table looks more balanced but other then that I think it looks really good now

Reply:
The columns have been swapped!
![image](https://user-images.githubusercontent.com/49821332/127238571-f078aa1f-a95f-4bc7-8ef3-7f0e228cc181.png)

Reply:
Thanks for the quick update!